tools/docker/Dockerfile-inclavare-dev-centos8.2

```diff
@@ -59,7 +59,7 @@ RUN rm -rf /etc/crictl.yaml && \
 FROM builder
 
 # install LVI binutils for rats-tls build
-RUN wget https://download.01.org/intel-sgx/sgx-linux/2.13.3/as.ld.objdump.gold.r3.tar.gz && \
+RUN wget https://download.01.org/intel-sgx/sgx-linux/2.14/as.ld.objdump.gold.r3.tar.gz && \
```

Comment:
Beside documents, it is better to define variables used to record the sdk version and release number to minimal your effort when updating sgx sdk.

ENV SGX_SDK_VERSION=2.14
ENV SGX_SDK_RELEASE_NUMBER=${SGX_SDK_VERSION}.100.2